tools/scripts/rsyslog-4xx-recovery

```diff
@@ -31,23 +29,6 @@ def main():
         except ValueError as e:
             write_stderr(str(e))
 
-        # now decide what do to based on eventnames
-        if headers["eventname"] == "PROCESS_STATE_FATAL":
-            headers.update(
-                dict(
-                    [x.split(":") for x in sys.stdin.read(int(headers["len"])).split()]
-                )
-            )
-
-            try:
-                # incoming event that produced PROCESS_STATE_FATAL will have a PID. SIGTERM it!
-                write_stderr(
-                    f"{datetime.datetime.now(timezone.utc)} - sending SIGTERM to proc={headers} with data={headers}\n"
-                )
-                os.kill(headers["pid"], signal.SIGTERM)
```

Comment:
A `PROCESS_STATE_FATAL` event is supervisor telling us that it has given up starting our process and will not try to start it again. There will be no `pid` in the header.